src/app/components/home/index.js

```diff
@@ -1,209 +1,229 @@
-  },
+  }
+
   componentDidMount() {
-    this.setupScrollMagic();
-    this.animTimeout = setTimeout(() => {
-      this.animateChevron();
-      this.setState({
-        chevronLoaded: true
-      })
-    }, 1500);
-  },
+
+    /*
+      Sets home-loader to be seen for a fixed time
+      TODO: Work out strategy here. It is a loader but ALSO an intro animation. We will want a minimum
+      time for it to be seen but also allow for the video and home content to load before we hide it.
+    */
+    setTimeout(() => {
+      this.setState({ contentLoaded: true });
+    }.bind(this), 4000);
+
+    /* Make sure that if the viewport is resized we update accordingly othewise scrolls/mousePositions will be out of sync */
+    window.addEventListener('resize', this.getViewportDimensions());
+
+    /*
+      The following are calculations for the ventures (dark background) section so we know when it
+      comes into view and we can show/hide it accordingly.
+    */
+    this.venturesHeight = this.venturesWrapper.getBoundingClientRect().height;
+    this.venturesPositionFromTop = this.venturesWrapper.offsetTop - (this.venturesHeight * 0.25);
+    const whereIsVentures = {
+      from: this.venturesPositionFromTop,
+      to: this.venturesPositionFromTop + this.venturesHeight
+    }
+    Flux.whereIsVentures(whereIsVentures);
+  }
+
   componentWillUnmount() {
-    this.teardownScrollMagic();
-    clearTimeout(this.animTimeout);
-  },
+    window.removeEventListener('resize', this.getViewportDimensions());
+  }
+
   render() {
-    const { page } = this.props;
-    const classes = classnames('page-home', this.props.className);
-    const featuredImage = getFeaturedImage(page);
-    // Show only the final frame of the Chevron animation on mobile
-    let Chevron;
-    if (window.innerWidth <= 480) {
-      Chevron = (<div className="down-chevron">
-        <svg ref="animsvg" title="Down arrow" role="img" viewBox="0 0 400 200"><g>
-        <path d="M195.864 143.667c19.556-14.667 39.556-28.89 59.11-43.556 2.224 2.67 6.224 8 8.446 10.67-22.222 16.89-45.778 32.45-67.556 50.67-21.778-17.78-44.89-33.33-67.11-50.22 2.22-2.66 6.22-8 8-11.11 20 14.67 39.555 29.33 59.11 43.56z"/>
-      </g></svg></div>);
-    } else {
-      Chevron = <DownChevron ref="downChevron" onClick={this.onClickDownChevron} customClass={this.state.chevronLoaded ? 'loaded' : ''} />;
+    const classes = classnames('page-home', this.props.className, {
+      /* venturesActive shows or hides the dark background depending on when it falls in/out of view */
+      venturesActive: this.props.documentScrollPosition > this.venturesPositionFromTop && this.props.documentScrollPosition < this.venturesPositionFromTop + this.venturesHeight,
+      /* when 'loaded', hide home-loader */
+      loaded: this.state.contentLoaded
+    });
+
+    /*
+      Text block contents
+      TODO: Do this nicer! Extract content. Perhaps when/if we integrate with CMS.
+    */
+    const textBlockIntro = {
```

Comment:
Although it is a temporary thing, I think you would benefit from specialising a component for each one of them, something on the lines of

```js
<Intro documentScrollPosition={this.props.documentScrollPosition}
           viewportDimensions={this.state.viewportDimensions} />
```

So abstracting `ScrollWrapper` and the hardcoded text as well.  Again, this is just to get a smaller home file (smaller render function too) to make it easier to reason about.